doc/ref_arch/openstack/README.md

```diff
@@ -37,8 +37,8 @@ This is OpenStack based Reference Architecture (RA-1)
 ## Table of Contents
 * [Chapter 01 - Overview](chapters/chapter01.md)
 * [Chapter 02 - Architecture Requirements](chapters/chapter02.md)
-* [Chapter 03 - NFVI and VIM Architecture](chapters/chapter03.md)
-* [Chapter 04 - NFVI and VIM Component Level Architecture](chapters/chapter04.md)
+* [Chapter 03 - Cloud Infrastructure Architecture](chapters/chapter03.md)
+* [Chapter 04 - Cloud Infrastructure and VIM Component Level Architecture](chapters/chapter04.md)
```

Comment:
Why is 'NFVI and VIM' replaced with 'Cloud Infrastructure' in Chapter 03 and only 'NFVI' and not 'NFVI and VIM' in Chapter 04?

Reply:
Looks like it's just to match the titles in the document, should be resolved separately from this PR.